tests/integration/helpers.py

```diff
@@ -51,14 +51,13 @@ async def install_plugins(
     assert res.status_code == 200, "Failed to request plugins install"
 
     # block until the UI does not have "Pending" in download progress column.
-    await unit.model.block_until(
+    await wait_for(
```

Comment:
`model.block_until` causes a lot of issues with websocket.disconnect errors. This juju interaction is extremely flaky and the custom `wait_for` should be used whenever applicable.